modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/exportimport/data/handler/DDLDisplayPortletDataHandler.java

```diff
@@ -51,12 +57,37 @@ public String getServiceName() {
 		return DDLConstants.SERVICE_NAME;
 	}
 
+	@Override
+	public boolean isDisplayPortlet() {
+		return false;
```

Comment:
Unique Integration [failure](https://testray.liferay.com/reports/production/logs/2023-07/test-1-25/test-portal-acceptance-pullrequest(master)/2179/modules-integration-mysql57-jdk8/0/1/jenkins-console.txt.gz?authuser=0):
` [exec] com.liferay.dynamic.data.lists.internal.exportimport.data.handler.test.DDLDisplayPortletDataHandlerTest > testIsDisplayPortlet FAILED
     [exec]     java.lang.AssertionError: expected:<true> but was:<false>`

Hi @bakayattila and @vendeltoreki,
Could you please take a look? Thx